docs/admin/prometheus.md

```diff
@@ -0,0 +1,39 @@
+# Prometheus
+
+Coder exposes many metrics which can be consumed by a Prometheus server, and give insight into the current state of a live Coder deployment.
+
+If you don't have an Prometheus server installed, you can follow the Prometheus [Getting started](https://prometheus.io/docs/prometheus/latest/getting_started/) guide.
+
+## Enable Prometheus metrics
+
+Coder server exports metrics via the HTTP endpoint, which can be enabled using either the environment variable `CODER_PROMETHEUS_ENABLE` or the flag `--prometheus-enable`.
+
+The Prometheus endpoint address is `http://localhost:2112/` by default. You can use either the environment variable `CODER_PROMETHEUS_ADDRESS` or the flag ` --prometheus-address <network-interface>:<port>` to select a different listen address.
```

Comment:
Sadly Prometheus doesn't work on our helm chart, which is our primary enterprise use case. Can we document this shortcoming, or ideally fix this? https://github.com/coder/coder/issues/4666

Reply:
Ok, I added a dedicated notice to document this inconvenience. I can take a look at fixing this later. I'm not sure if it isn't a rabbit hole :)